tomviz/tomvizPythonConfig.h.in

```diff
@@ -91,6 +93,10 @@ void PythonAppInitPrependPathLinux(const std::string& exe_dir)
     PythonAppInitPrependPythonPath(exe_dir + "/../" TOMVIZ_PYTHON_INSTALL_DIR);
 
     // we don't add anything for ParaView since, ParaView takes care of that.
```

Comment:
This begs the question of why ParaView is not taking care of this on Linux. If it is not (but is on other platforms) we should file a bug, and change this comment accordingly.